static/main.css

```diff
@@ -472,7 +463,7 @@ body.drag .app::after {
 }
 
 .torrent-details {
-  padding: 8em 0 20px 0;
```

Comment:
this was hack used to fix the positioning layout break, right now I removed the absolute position and used flex and enhanced the layout.